packages/vue/src/components/organisms/SfMegaMenu/_internal/SfMegaMenuColumn.html

```diff
@@ -0,0 +1,8 @@
+<div v-if="isActive" class="sf-mega-menu-column">
+    <!-- @slot Custom title markup -->
+    <slot name="title" v-bind="{ title }">
+      <h3 class="sf-mega-menu-column__title">{{ title }}</h3>
+    </slot>
+    <!-- @slot Custom column markup -->
+    <slot />
```

Comment:
Isn't this a default behavior to fill this slot? Why it's labeled as "custom"?

Reply:
fixed: https://github.com/DivanteLtd/storefront-ui/pull/472/commits/6bb9a071b4e0209037779759e384dfdec4d3ad96